Cargo.toml

```diff
@@ -74,6 +75,7 @@ tracing-subscriber = { version = "0.3", features = [
     "local-time",
 ] }
 url = "2.5.4"
+atty = "0.2"
```

Comment:
`atty` is unmaintained, [std.io.IsTerminal](https://doc.rust-lang.org/std/io/trait.IsTerminal.html) is in std now